### PR TITLE
iucode-tool: fix host-compile on macos and non-x86 linux

### PR DESCRIPTION
**Pull request by svlobanov**
This PR has two commits:
1. add argp-standalone/host required to build iucode-tool on macos
2. cpuid.h fix for iucode-tool on non-x86 linux and macos build hosts

Detailed description is provided in the commits.

### CODE REVIEW

File: package/libs/argp-standalone/Makefile [hunk 1]
```diff
@@ -45,4 +46,14 @@ define Build/InstallDev
...
+	$(INSTALL_DIR) $(1)/include
+	$(CP)   $(HOST_BUILD_DIR)/argp.h \
+		$(1)/include/
+	$(INSTALL_DIR) $(1)/lib
+	$(CP)   $(HOST_BUILD_DIR)/libargp.a \
+		$(1)/lib/
+endef
+
 $(eval $(call BuildPackage,argp-standalone))
+$(eval $(call HostBuild))
```
Comment (neheb) [inline package/libs/argp-standalone/Makefile:59]: No newline?

Comment (svlobanov) [reply package/libs/argp-standalone/Makefile:59]: yes, no newline. Is it mandatory? does OpenWrt have guidelines related to this?

Comment (neheb) [reply package/libs/argp-standalone/Makefile:59]: It's best practice. Both vim and nano automatically add one by default.

Comment (svlobanov) [reply package/libs/argp-standalone/Makefile:59]: ok, I will this no newline issue

Comment (svlobanov) [reply package/libs/argp-standalone/Makefile]: @neheb 'no newline' issue is fixed

**Comment by chunkeey** [comment]
Thank you! Merged.

**Pull request closed without merging**